geniza/corpus/admin.py

```diff
@@ -462,13 +463,48 @@ def get_urls(self):
     actions = (export_to_csv,)
 
 
+@admin.register(DocumentNeedsReview)
+class DocumentNeedsReviewAdmin(DocumentAdmin):
+    ordering = ("needs_review",)
+
```

Comment:
Should probably turn off the needs review filter that we're inheriting from `DocumentAdmin`

Reply:
I thought about it, but there's a lot to be copied. Are we okay having this twice, minus `needs_review`?

```
list_filter = (
        "doctype",
        (
            "footnotes__content",
            custom_empty_field_list_filter(
                "transcription", "Has transcription", "No transcription"
            ),
        ),
        (
            "textblock__fragment__iiif_url",
            custom_empty_field_list_filter("IIIF image", "Has image", "No image"),
        ),
        (
            "needs_review",
            custom_empty_field_list_filter("review status", "Needs review", "OK"),
        ),
        "status",
        ("textblock__fragment__collection", admin.RelatedOnlyFieldListFilter),
        ("languages", admin.RelatedOnlyFieldListFilter),
        ("secondary_languages", admin.RelatedOnlyFieldListFilter),
    )
```

Reply:
No, I don't want it copied. Maybe we could make a copy of the list and remove the filter we don't want, but perhaps  we should just leave it as is for first round testing and let the team tell us what filters they need.